stable/kibana/templates/ingress.yaml

```diff
@@ -1,7 +1,11 @@
 {{- if .Values.ingress.enabled -}}
 {{- $serviceName := include "kibana.fullname" . -}}
 {{- $servicePort := .Values.service.externalPort -}}
+{{- if semverCompare "> 1.19" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Sorry, for using an example version here.

New api will work form Kubernetes 1.14: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#deprecations

So it should be ">= 1.14"